core/src/main/java/nu/yona/server/subscriptions/entities/UserPrivate.java

```diff
@@ -162,4 +163,9 @@ public UUID getUserAnonymizedID()
 	{
 		return userAnonymizedID;
 	}
+
+	public Set<Buddy> getBuddiesOfRemovedUsers()
```

Comment:
I don't think present continuous is necessary here. Common English is "related to".